src/install/kubernetes/guided/configuration-guided.mdx

```diff
@@ -0,0 +1,40 @@
+---
+componentType: default
+headingText: Gather log data
```

Comment:
@nbaenam sorry for the silly question, but is this saying that the guided install will set up log forwarding similar to how the Helm chart installation will set up Fluent Bit log forwarding?

Reply:
Hey @mangulonr, could you please clarify this a little bit? Thanks!

Reply:
Hi @rhetoric101, this is the answer from @mangulonr:
`The defaults are the same regardless of the installation method.`